assets/scss/components/stencil/heroCarousel/_heroCarousel.scss

```diff
@@ -4,42 +4,36 @@
 // Purpose: Styles the hero carousel component, which basically adds extras to the
 // vendor slick carousel, to specifically display hero images.
 //
-// 1. Hide the actual image element to allow background-size: cover treatment on
-//    large screens to scale image. Use visibility because the JS uses the height
-//    to set the slide size. (For browsers with object-fit only)
+// 1. Use object-fit on the image tag to keep aspect ratio and cover the space
+//    created by the outer div. The JS will detect browsers that do not support
+//    object-fit and instead copy the image src to a the background-image of the
+//    wrapper div and add the compat-object-fit class.
 //
-// 2. With JS on, minimise the jump in content as you progressively enhance the
-//    slider. Slowly fade and slide the height, instead of a jump in collapsing
-//    hidden content.
-//
-// 3. Visually overrides the top margin for '.body' defined in _body.scss.
+// 2. Visually overrides the top margin for '.body' defined in _body.scss.
 //    The '.body' top margin creates space between the header and page content.
 //    However, on the homepage, we want the carousel to be flush with the header.
 //
-// 4. Allows image to scale on large screens while preventing the top and bottom
+// 3. Allows image to scale on large screens while preventing the top and bottom
 //    from becoming cut off.
 // -----------------------------------------------------------------------------
 
 .heroCarousel {
-    margin-bottom: (spacing("double") + spacing("single"));
+    margin-bottom: 30px; // from slick-theme.scss
```

Comment:
🍹 no `remCalc`?

Reply:
Reverting this change, it is unnecessary.